### PR TITLE
Add uuid field to AnimationClip.toJSON

### PR DESCRIPTION
**Pull request by etr2460**
When converting the scene to JSON all objects except animation clips have uuid fields included. This change adds the uuid to the JSON object for AnimationClips.

### CODE REVIEW

**Comment by Mugen87** [comment]
I guess this PR is only consistent if `ObjectLoader` also restores the serialized uuid. This means 
a change in this method.
https://github.com/mrdoob/three.js/blob/ec67749b0767e2727deebd52da2dbd3a4e7ab6a2/src/loaders/ObjectLoader.js#L453-L467

**Comment by etr2460** [comment]
Updated with a change to AnimationClip.parse() and the AnimationClip constructor

**Comment by etr2460** [comment]
I felt a little weird about overwriting the uuid field after the object was already constructed, but it works for me if it's preferred!

**Comment by Mugen87** [comment]
`ObjectLoader` already does this at different places e.g.

https://github.com/etr2460/three.js/blob/232586ee65f5951957efef4050fefeaef9664a36/src/loaders/ObjectLoader.js#L554

File: src/loaders/ObjectLoader.js [hunk 1]
```diff
@@ -458,6 +458,12 @@ Object.assign( ObjectLoader.prototype, {
 
 			var clip = AnimationClip.parse( json[ i ] );
 
+			if ( json[ i ].uuid !== undefined ) {
+
+				clip.uuid = json[ i ].uuid;
+
+			}
+
```
Comment (Mugen87) [inline src/loaders/ObjectLoader.js]: Please write the method like this now:
```js
parseAnimations: function ( json ) {

	var animations = [];

	for ( var i = 0; i < json.length; i ++ ) {

		var data = json[ i ];

		var clip = AnimationClip.parse( data );

		if ( data.uuid !== undefined ) clip.uuid = data.uuid;

		animations.push( clip );

	}

	return animations;

},
```
In this way it's more similar to others in `ObjectLoader`.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**